src/2ls/types.h

```diff
@@ -0,0 +1,17 @@
+/*******************************************************************\
+
+Module: Type declarations
```

Comment:
:pick: I'd call this "traces.h"

Reply:
Sure, probably makes sense. I wasn't very keen on creating a separate file for this anyway but it is required by multiple modules and putting it in one of them resulted in circular includes. But lets keep the file for traces only to avoid putting together things which don't belong together :+1: